examples/newrelic/newrelic-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: newrelic
```

Comment:
Please don't use "name" as a label key. A number of users have found that confusing in the past and we're trying to eliminate uses of it in our examples.